src/test/fixtures/sources.ts

```diff
@@ -1,15 +1,21 @@
-export const mockSourcesByProvider = (provider) => {
-  if (provider === 'aws') {
-    return [
+import {
+  GetSourceListApiResponse,
+  GetSourceUploadInfoApiResponse,
+} from '../../store/provisioningApi';
+
+export const mockSourcesByProvider = (
+  provider: string
+): GetSourceListApiResponse => {
+  const mockSources = {
```

Comment:
Same comment as below, what if we type `mockSources`, get rid of the explicit return type and let Typescript infer it, and remove `as keyof typeof mockSources` in the return statement?